src/spec-configuration/containerFeaturesOCI.ts

```diff
@@ -60,34 +60,34 @@ export function getOCIFeatureSet(output: Log, identifier: string, options: boole
     return featureSet;
 }
 
-export function getFeatureRef(output: Log, identifier: string): OCIFeatureRef {
+export function getFeatureRef(output: Log, resourceAndVersion: string): OCIFeatureRef {
 
     // ex: ghcr.io/codspace/features/ruby:1
-    const splitOnColon = identifier.split(':');
-    const id = splitOnColon[0];
+    const splitOnColon = resourceAndVersion.split(':');
+    const resource = splitOnColon[0];
     const version = splitOnColon[1] ? splitOnColon[1] : 'latest';
```

Comment:
Suggestion: cleaner code

`const version = splitOnColon[1] ?? 'latest';`

Reply:
definitely, for another time 😅